integreat_cms/cms/templatetags/base_filters.py

```diff
@@ -35,3 +36,22 @@ def get_private_member(element, key):
     :rtype: object
     """
     return element[f"_{key}"]
+
+
+@register.simple_tag
+def get_mt_visibility(region, perms):
+    """
+    This tag checks whether either DEEPL or SUMM.AI is activated in the region and the user has permission to manage automatic translations.
+
+    :param region: The rcurrent region
+    :type region: ~integreat_cms.cms.models.regions.region.Region
+
+    :param perms: Permission of the user
+    :type perms: ~django.contrib.auth.context_processors.PermWrapper
```

Comment:
Could you add this to Sphinx' nitpicky ignore list?

Reply:
Applied 👍 Thank you!